clients/go/pkg/zbc/client.go

```diff
@@ -82,7 +82,7 @@ func (c *ClientImpl) NewTopologyCommand() *commands.TopologyCommand {
 }
 
 func (c *ClientImpl) NewDeployProcessCommand() *commands.DeployCommand {
-	return commands.NewDeployCommand(c.gateway, c.credentialsProvider.ShouldRetryRequest)
+	return commands.NewDeployCommand(c.gateway, c.credentialsProvider.ShouldRetryRequest) // nolint
```

Comment:
❓ Should we log a warning when this is used as an additional way to inform users about deprecation for removal? This is something we've seen in elastic search AFAICR.

Reply:
I hate that Elastic does that actually :smile: It's even worse in their case, as the client receives a warning about server configuration.

I don't think this is common practice in Go, and we don't do this for the Java client either. GoLand, for example, will highlight usages of deprecated code (much like IntelliJ with Java). I would leave it as is.